docs/source/reference/storage.rst

```diff
@@ -244,7 +244,21 @@ To manage persistent Storage objects, the sky CLI provides two useful commands -
     NAME               CREATED     STORE  COMMAND                                        STATUS
     sky-dataset-romil  3 mins ago  S3     sky launch -c demo examples/storage_demo.yaml  READY
 
-2.  :code:`sky storage delete` allows you to delete any Storage objects managed
+2.  :code:`sky storage create` allows you to create a sky Storage object. Common use cases include:
+   - Create a storage object by uploading contents from local source path to specified stores
+        :code:`$ sky storage create imagenet -src /Users/skypilotUser/Project -s S3 -s GCS`
+   - Create an empty storage object with GCS stores
+        :code:`$ sky storage create emptyStore -s GCS`
+   - Create a storage object by uploading local source contents to default S3 store
+        :code:`$ sky storage create cleanedData /Users/skypilotUser/Project/CleanedData`
```

Comment:
`--source` needs to be specified?

Reply:
If source isn't specified, it creates an empty Sky Storage object through the requested cloud provider.